### PR TITLE
tools: not warning about deprecated declarations

### PR DESCRIPTION
**Pull request by alyssawilk**
Also removing a couple of unnecessary validate.proto files.
This removes several thousands of lines worth of warnings from our build logs.

### CODE REVIEW

File: .bazelrc [hunk 1]
```diff
@@ -41,6 +41,7 @@ build --action_env=CC
 build --action_env=CXX
 build --action_env=LLVM_CONFIG
 build --action_env=PATH
+build --copt=-Wno-deprecated-declarations
```
Comment (lizan) [inline .bazelrc:44]: this need to be in `build:linux` block above to not break windows.

**Comment by phlax** [comment]
if only we could find a solution to https://github.com/bazelbuild/bazel/issues/7157 - we could really clean up ci logs

**Comment by alyssawilk** [comment]
oh thanks for the link @phlax  - I'll ping internally and see if I can get any traction.

**Comment by alyssawilk** [comment]
lizan anything else you'd like to see here?

**Review by lizan — APPROVED** [review]

**Pull request merged**